src/strings.jl

```diff
@@ -123,3 +123,33 @@ function parseerror()
   Please make sure that the string follows any of the following Well-Known-Text formats: OGC WKT1, ESRI WKT1, WKT2.
   """))
 end
+
+
+function _init_epsg_datadep_registration()
+  epsg_dataset_metadata = DataDep(
+    "epsg",
+
+    # TODO: do we want this proper message here or just auto download?
```

Comment:
Could you please clarify what do you mean by auto download? The default message generated by DataDeps.jl?

Reply:
I meant do we want to wait for the user to approve the (y/n) prompt (default in DataDeps) or is this such a small dataset that this is not needed?

EDIT: oh dang! I got duped by not pressing submit review on github.